values.yaml

```diff
@@ -440,6 +441,11 @@ rabbitmq-ha:
   rabbitmqUsername: admin
   # TODO: Use default random 24 character password, but need to fetch this string for use by downstream services
   rabbitmqPassword: 9jS+w1u07NbHtZke1m+jW4Cj
+  # RabbitMQ available memory
+  rabbitmqMemoryHighWatermark: "0.8"
+  rabbitmqMemoryHighWatermarkType: relative
+  # Up to 255 character string
+  rabbitmqErlangCookie: 8MrqQdCQ6AQ8U3MacSubHE5RqkSfvNaRHzvxuFcG
```

Comment:
It's questionable to generate the key for everyone.

I realize that it could be useful for someone re-deploying (destroying/creating) rabbitmq-ha for many times in a row. And it also could take time to understand they need the same `rabbitmqErlangCookie` to make re-deployment not fail.

Instead of forcing this rabbitmq cookie for everyone, please hide the value under comment and include recommendation in comment about why it could be useful or important. This way someone experiencing re-deployment issue would be able to consult with this Helm values hint.

Reply:
Agree on that, will solve in the next commit.